app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,120 @@
     <string name="app_name">Cyrcle</string>
     <string name="default_web_client_id">836297757379-3rvkggdu23fdvnof5nko2lablc993e0u.apps.googleusercontent.com</string>
 
+    <!-- Common section -->
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+    <string name="no_selected_parking_error">No parking selected. Should not happen</string>
+    <string name="default_parking_name">Unnamed Parking</string>
+    <string name="distance_km">%.2f km</string>
+    <string name="distance_m">%.0f m</string>
+    <string name="no_reviews">No reviews yet</string>
+    <plurals name="reviews_count">
+        <item quantity="zero">(0 review)</item>
+        <item quantity="one">(1 review)</item>
+        <item quantity="other">(%d reviews)</item>
+    </plurals>
+
+    <!-- Attributes Picker -->
+    <string name="attributes_picker_title_label">Title</string>
+    <string name="attributes_picker_protection_label">How protected from the weather is the spot?</string>
+    <string name="attributes_picker_capacity_label">How many bikes can we park?</string>
+    <string name="attributes_picker_rack_type_label">Which type of rack is in the parking spot?</string>
+    <string name="attributes_picker_has_surveillance_question">Is the parking spot under camera surveillance ?</string>
```

Comment:
Here you are defining the string name as `attributes_picker_has_surveillance_question` but the other strings name do not finish with question despite the fact that they are.

Reply:
and there is a space between `e` and `?` but that is not so important

Reply:
Indeed, because `_had_surveillance_question` is the text for an `BooleanRadioButton(question)`, whereas the three others are texts for a `DropDownMenu(label)`so maybe the right course of actions for this is to simplify the labels of the dropdown menu to `Capacity` (for example), as this will also reduce the text weight on a screen.